servers/dsub/jobs/controllers/utils/jobs_generator.py

```diff
@@ -95,7 +95,11 @@ def generate_jobs_by_window(provider, project_id, window_min, window_max=None):
     Returns:
         generator: Retrieved jobs
     """
-    create_time_min = window_min - datetime.timedelta(days=_MAX_RUNTIME_DAYS)
+    create_time_min = None
+
+    if window_min is not None:
+        create_time_min = window_min - datetime.timedelta(
+            days=_MAX_RUNTIME_DAYS) if window_min is not None else None
```

Comment:
window_min is already not None at this point in the code

Reply:
Fixed.